### ISSUE TITLE
Land strings for Accounts & Receive Tab

### ISSUE THREAD

**Issue by athomasmoz** [mozilla-mobile/firefox-tv#2643]
### Why/User Benefit/User Problem

This story is to land our strings for Accounts & Receive tab so that localization can start. Strings will not be visible to users until we launch the new features

### Acceptance Criteria (how do I know when I’m done?)

- [x] Final strings have been landed
- [x] All strings have been sent for localization ([l10n pr](https://github.com/mozilla-l10n/android-l10n/pull/63))

### Finalized Strings

https://docs.google.com/spreadsheets/d/1z852T9AKhFpQFq8JcX633-zsC81vxAkKYKyilwdistU/edit?ts=5d423a76#gid=0

**Comment by liuche** [comment]
I think we should pre-land these strings and export them for L10N, and leave off the one string that we're not sure about. I've marked the one w/ Firefox Preview in the spreadsheet.

**Comment by athomasmoz** [comment]
Thanks, and when we launch it will still be Preview, so we're safe to use that.

**Comment by liuche** [comment]
Let's try to land these by 8/9 and exported so L10N has enough time to localized for us.

**Comment by mcomella** [comment]
@dnarcese Do you know if we need to run a manual export (using the old system) on our strings or if l10n will automate the export (with the new system)? Can you summarize the current state of the world? As per liuche's comment, we'd like to export these strings ASAP so I'm hoping something is in place. :)

CC @flodolo

**Comment by flodolo** [comment]
Redirecting to @pike who's covering for @Delphine this week

**Comment by Pike** [comment]
We started with `android-l10n` in https://github.com/mozilla-l10n/android-l10n/pull/59.

I'll create the first import back from the strings in a moment.

**Comment by Pike** [comment]
PR #2673 is up :-)

**Comment by dnarcese** [comment]
@mcomella We just need to land strings on master, and l10n team will do regular exports to android-l10n repo.

**Comment by dnarcese** [comment]
l10n does not want to rush merging these in on a Friday (it is late night their time) - it will get merged on Monday (cc @liuche)

**Comment by dnarcese** [comment]
All strings have now been exported.
